easybuild/easyconfigs/v/VASP/VASP-5.4.1-intel-2016.02-GCC-4.9.eb

```diff
@@ -14,15 +14,26 @@ toolchainopts = {'usempi': True}
 # Vasp is proprietary software, see http://www.vasp.at/index.php/faqs on how to get access to the code
 sources = [SOURCELOWER_TAR_GZ]
 
+patches = ['patch.5.4.1.08072015', 'patch.5.4.1.27082015', 'patch.5.4.1.06112015']
+
+checksums = ['57c2b07d0f70987000033984e913f1a6',  # vasp.5.4.1.tar.gz
+             'ee242452bba6bb37459745f4617649d7',  # patch.5.4.1.08072015
+             'b137eb13804a60df382e77495dc738f9',  # patch.5.4.1.27082015
+             'd64b8f0ae861db7a76e6e9c9ffa188a1',  # patch.5.4.1.06112015
+             ]
+
 prebuildopts = 'cp arch/makefile.include.linux_intel ./makefile.include && '
 
 # path to libfftw3xf_intel.a is hardcoded in makefile.include
 prebuildopts += 'sed -i "s|\$(MKLROOT)/interfaces/fftw3xf|\$(FFTW_LIB_DIR)|" makefile.include && '
 
+# we use static mkl later
```

Comment:
I think we better make clear *why* we do this here; dynamic linking to MKL is a cause of problems, and it's not clear where the static linking is taken care of (I guess it's via `$LIBSCALAPACK`?)